content/cn/docs/admin/authorization/_index.md

```diff
@@ -31,18 +31,18 @@ Kubernetes 使用 API​​ 服务器授权 API 请求。它根据所有策略
 
 Kubernetes 仅查看以下API请求属性:
 
-* **user**  - 验证期间提供的 `user` 字符串
-* **group**  - 认证用户所属的组名列表
-* **“extra"**  - 由认证层提供的任意字符串键到字符串值的映射
-* **API**  - 指示请求是否用于API资源
-* **Request path**  - 诸如`/api`或`/healthz`的其他非资源端点的路径(请参阅[kubectl](#kubectl)).
-* **API request verb**  -  API 动词 `get`，`list`，`create`，`update`，`patch`，`watch`，`proxy`，`redirect`，`delete`和`deletecollection`用于资源请求。要确定资源 API 端点的请求动词，请参阅**确定下面的请求动词**.
-* **HTTP request verb**  -  HTTP动词`get`，`post`，`put`和`delete`用于非资源请求
-* **Resource**  - 正在访问的资源的ID或名称(仅适用于资源请求)
+* **user**  - 验证期间提供的 `user` 字符串。
```

Comment:
I'd suggest we remove all the '。' in this list because '。' only makes sense when there is a complete sentence.